examples/zephyr/location/wifi/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
It would be good to have a `README.md` for this example, especially given that this service is currently in private access. An explanation regarding the Wi-Fi playback functionality would likely also be useful 👍🏻